src/Shared/HttpSys/RequestProcessing/PathNormalizer.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Diagnostics;
+
```

Comment:
Not quite a noop merge but there should be a new line anyway.

Reply:
@JunTaoLuo :ok:, you decided to revert most changes to this file. Why is it different in 2.1?

Reply:
The 2.2 change takes advantage of a C# change that isn't available in 2.1. The 3.0 code also looks similar to the 2.2 code here.